apt/minimization/minimizer.py

```diff
@@ -70,11 +74,14 @@ class GeneralizeToRepresentative(BaseEstimator, MetaEstimatorMixin, TransformerM
     """
 
     def __init__(self, estimator=None, target_accuracy=0.998, features=None,
-                 cells=None):
+                 cells=None, categorical_features=None):
```

Comment:
Please add a description of the categorical_features param to the docstring of the class